ComplianceAsCode/content
# Fix #11902 issue

#### Description:

On Ubuntu 22.04 pam_tally2 was replaced with pam_faillock and checks should be done on files referenced by the CIS guide:

```
-w /var/log/lastlog -p wa -k logins
-w /var/run/faillock -p wa -k logins
```

#### Rationale:

- Fixes #11902

#### Review Hints:

Build the product:

```
./build_product ubuntu2204
```

To test these changes with Ansible:

```
ansible-playbook build/ansible/ubuntu2204-playbook-cis_level2_server.yml --tags "audit_rules_login_events_faillock"
```

To test changes with bash, run the remediation sections: `xccdf_org.ssgproject.content_rule_audit_rules_login_events_faillock`

Expected output:

```
Title   Record Attempts to Alter Logon and Logout Events - faillock
Rule    xccdf_org.ssgproject.content_rule_audit_rules_login_events_faillock
Result  pass
```
